### ISSUE TITLE
Can I tag the openpose dataset myself?

### ISSUE THREAD

**Issue by CG66** [CMU-Perceptual-Computing-Lab/openpose_train#13]
Openpose is usually used to identify human joints，but if  i want to identify other animals，how should I label the image
 I am searching for a long time on net. But no use. Please help or try to give some ideas how to achieve this.

**Comment by gineshidalgo99** [comment]
Long story short: Create and label a huge dataset of keypoints for that specific animal, create a skeleton for that structure analog to the one we created for humans, and train any pose detector for that.

I believe there are some datasets, but too small to train this kind of detector.

**Comment by CG66** [comment]
Thank you for your reply
I usually use labelme for tags, but it seems that I can't use it for key node tags. Openpose seems to need heat map for key node tags. What labeling tools do you know that meet the requirements? Or did you use the ready-made label directly or type the label yourself? Thank you again







------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Gines"<notifications@github.com&gt;;
发送时间:&nbsp;2019年11月11日(星期一) 下午2:36
收件人:&nbsp;"CMU-Perceptual-Computing-Lab/openpose_train"<openpose_train@noreply.github.com&gt;;
抄送:&nbsp;"哼哼公子"<841102344@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [CMU-Perceptual-Computing-Lab/openpose_train] Can I tag the openpose dataset myself? (#13)




Long story short: Create and label a huge dataset of keypoints for that specific animal, create a skeleton for that structure analog to the one we created for humans, and train any pose detector for that.
 
I believe there are some datasets, but too small to train this kind of detector.

**Comment by gineshidalgo99** [comment]
I'd recommend you to check the C++ data loader code and the COCO format. It only takes as input the body keypoints (17 keypoints / person). See how it works for body, it's not something I can do a tutorial (too long to explain), it's faster to just understand the code.
